### PR TITLE
Logging changes according to new logging framework

### PR DESCRIPTION
**Pull request by anmolsachan**
tendrl-bug-id: Tendrl/specifications#28
tendrl-spec: Tendrl/specifications#127
reference-spec: Tendrl/specifications#94
reference-code-pr: Tendrl/commons#161

Signed-off-by: anmolsachan asachan@redhat.com

### CODE REVIEW

File: tendrl/commons/etcdobj/__init__.py [hunk 1]
```diff
@@ -142,7 +155,7 @@ def __init__(self, **kwargs):  # pragma: no cover
         for key in dir(self):
             if not key.startswith('_'):
                 attr = getattr(self, key)
-                if issubclass(attr.__class__, Field):
+                if issubclass(attr.__class__, fields.Field):
```
Comment (anmolsachan) [inline tendrl/commons/etcdobj/__init__.py]: @r0h4n I was getting error here while testing, so I made this change. Please check if this is correct.

**Comment by anmolsachan** [comment]
@GowthamShanmugam @shtripat please review

File: tendrl/commons/config.py [hunk 2]
```diff
@@ -1,24 +1,23 @@
-import logging
 import os
-
+import sys
 import yaml
 
-LOG = logging.getLogger(__name__)
+from tendrl.commons.event import Event
```
Comment (shtripat) [inline tendrl/commons/config.py]: Are these used in this source?

File: tendrl/commons/etcdobj/__init__.py [hunk 3]
```diff
@@ -79,8 +79,21 @@ def save(self, obj):
...
+                    Message(
+                        priority="debug",
+                        publisher=tendrl_ns.publisher_id,
+                        payload={"message": "Writing %s to %s" %
+                                            (item['key'], item['value'])
+                                 }
+                    )
+                )   
+                self.client.write(item['key'], item['value'], quorum=True)
+            except AttributeError:
```
Comment (shtripat) [inline tendrl/commons/etcdobj/__init__.py]: Why few places we are using try-except? I feel you should add some comments in this cases..

File: tendrl/commons/manager/__init__.py [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
-import logging
 
 import six
 
-from tendrl.commons import jobs
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
 
-LOG = logging.getLogger(__name__)
+from tendrl.commons import jobs
```
Comment (shtripat) [inline tendrl/commons/manager/__init__.py]: Who is using this import

File: tendrl/commons/config.py (hunk 2 above)
Comment (anmolsachan) [inline tendrl/commons/config.py]: I will fix this

File: tendrl/commons/etcdobj/__init__.py (hunk 3 above)
Comment (anmolsachan) [inline tendrl/commons/etcdobj/__init__.py]: I will add the comments. Thank you for pointing.

File: tendrl/commons/manager/__init__.py (hunk 4 above)
Comment (anmolsachan) [inline tendrl/commons/manager/__init__.py]: I think it is being used in line 21.

Comment (shtripat) [inline tendrl/commons/manager/__init__.py]: ok

**Comment by anmolsachan** [comment]
Moved to #182

**Pull request closed without merging**